test/object-sanitizer.test.js

```diff
@@ -0,0 +1,62 @@
+/* global it, describe */
+'use strict'
+const expect = require('chai').expect
+const objectSanitizer = require('../lib/object-sanitizer')
+
+describe('objectSanitizer basic', function () {
+  // used in insert, find, count etc.
+  it('should sanitizes numbers', function () {
+    const obj = { a: 0, b: 12 }
+    const result = objectSanitizer.sanitize(obj)
+    expect(JSON.stringify(result)).equal(JSON.stringify({ a: '?', b: '?' }))
+  })
+
+  it('should sanitizes strings', function () {
+    const obj = { a: 'hide', b: 'me' }
+    const result = objectSanitizer.sanitize(obj)
+    expect(JSON.stringify(result)).equal(JSON.stringify({ a: '?', b: '?' }))
+  })
+
+  it('should sanitizes double quoted strings', function () {
+    const obj = { a: 'hide', b: 'me' }
```

Comment:
I think `obj` should be `{ a: "hide", b: "me" }` here, else I think this test is the same as the previous one.

Reply:
The linter fixed it.. :D
Will prevent all no linting comment.